### ISSUE TITLE
Preferences fields need inputTypes

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#202]
```
Certain of the fields in the preferences could do with an inputType. 
(http://developer.android.com/reference/android/widget/TextView.html#attr_androi
d:inputType)
This would benefit the user by providing the appropriate keyboard for entering 
certain inputs.

The ones I noticed were all in res/xml/prefs_connection.xml:

notifiedAddress is an email address, so it should have a 
android:inputType="textEmailAddress".
login maybe needs the email input type, too? I use apps, so it's an email 
address on mine, but I'm not sure it's appropriate in general.
serverHost, serviceName and mucServer are domains, so they should probably have 
android:inputType="textUri".
```

Original issue reported on code.google.com by `Eckankar` on 24 Jul 2011 at 2:24

**Comment by GoogleCodeExporter** [comment]
```
Good point, we'll incorporate that. Or maybe you want to supply a patch?
```

Original comment by `fschm...@gmail.com` on 24 Jul 2011 at 6:29
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Here's a patch of the proposed changes.

I do not currently have a functioning Android SDK handy, so this is untested.

Also, it does not include the inputType for the login field, as I'm unsure as 
to whether it should have the textEmailAddress type. Opinions?


```

Original comment by `Eckankar` on 24 Jul 2011 at 8:01

Attachments:
- [inputTypes.patch](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-202/comment-2/inputTypes.patch)

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 4796425e39ee.
```

Original comment by `fschm...@gmail.com` on 19 Aug 2011 at 9:57
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 19 Aug 2011 at 9:59
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:54
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 3.2
```

Original comment by `Florent....@gmail.com` on 28 Aug 2011 at 3:46
- Changed state: **Deployed**
